docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -226,74 +178,78 @@ def get_orbit_actor(orbit_points):
 uranus_track = []
 neptune_track = []
 
-##############################################################################
-# Define two new functions to use in ``timer_callback`` to update the planet
-# positions and their tracks to visualize their orbit: ``update_track`` and
-# ``update_planet_position``.
 
+##############################################################################
+# Define one new function to use in ``timer_callback`` to update the planet
+# positions ``update_planet_position``.
 
-def update_planet_position(r_planet, planet_actor, planet_track, cnt):
+def update_planet_position(r_planet, planet_actor, cnt):
     pos_planet = get_orbital_position(r_planet, cnt)
     planet_actor.SetPosition(pos_planet[0], 0, pos_planet[1])
-    planet_track.append([pos_planet[0], 0, pos_planet[1]])
     return pos_planet
 
 
-def update_track(positions_planet, planet_track, planet_orbit_actor):
-    positions_planet[:] = np.array(planet_track)
-    utils.update_actor(planet_orbit_actor)
-
-
 ##############################################################################
-# Define the ``timer_callback`` function, which controls what events happen
-# at certain times, using the counter. Redefine the position of each planet
-# actor using ``get_orbital_position,`` assigning the x and y values of
-# each planet's position with the newly calculated ones. Append each new
-# planet position to its corresponding track array.
+# ``calculate_path`` function is for calculating the path/orbit
+# of every planet.
 
-def timer_callback(_obj, _event):
-    cnt = next(counter)
-    showm.render()
+def calculate_path(r_planet, planet_track, cnt):
+    for i in range(cnt):
+        pos_planet = get_orbital_position(r_planet, i)
+        planet_track.append([pos_planet[0], 0, pos_planet[1]])
 
-    global mercury_track, venus_track, earth_track, mars_track, jupiter_track
-    global saturn_track, uranus_track, neptune_track
 
-    update_planet_position(r_mercury, mercury_actor, mercury_track, cnt)
+##############################################################################
+# First we are making two lists that will contain radiuses and track's lists.
+# and then we are calculating and updating the path/orbit
+# before animation starts.
 
-    update_planet_position(r_venus, venus_actor, venus_track, cnt)
+r_planets = [r_mercury, r_venus, r_earth, r_mars,
+             r_jupiter, r_saturn, r_uranus, r_neptune]
+planets_tracks = [mercury_track, venus_track, earth_track, mars_track,
+                  jupiter_track, saturn_track, uranus_track, neptune_track]
 
-    update_planet_position(r_earth, earth_actor, earth_track, cnt)
+for r_planet, planets_track in zip(r_planets, planets_tracks):
+    calculate_path(r_planet, planets_track, 2200)
```

Comment:
Since we change the equation 1000 is not enough to calculate the full path of many planets.
I will make that value variable, so it changes according to the radius of orbit of planet.